install_packages.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+# Script for installing packages and removing apt cache
+
+set -e
+set -u
+export DEBIAN_FRONTEND=noninteractive
+n=0
+max=2
+until [ $n -gt $max ]; do
+    set +e
+    (
+      apt-get update -qq &&
+      apt-get install -y --no-install-recommends "$@"
+    )
+    CODE=$?
+    set -e
+    if [ $CODE -eq 0 ]; then
+        break
+    fi
+    if [ $n -eq $max ]; then
+        exit $CODE
+    fi
+    echo "apt failed, retrying"
+    n=$(($n + 1))
+done
+rm -r /var/lib/apt/lists /var/cache/apt/archives
```

Comment:
tiniest of nits: something funny going on with the line ending here (missing LF at end of line?) — check your editor config?